src/PrestaShopBundle/Form/Admin/Type/ChangePasswordType.php

```diff
@@ -69,41 +68,89 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder
             ->add('change_password_button', ButtonType::class, [
-                'label' => false,
+                'label' => $this->trans('Change password...', [], 'Admin.Actions'),
+                'attr' => [
+                    'class' => 'btn-outline-secondary js-change-password',
+                ],
+            ])
+            ->add('old_password', PasswordType::class, [
+                'label' => $this->trans('Current password', [], 'Admin.Advparameters.Feature'),
+                'required' => true,
             ])
-            ->add('old_password', PasswordType::class)
             ->add('new_password', RepeatedType::class, [
                 'type' => PasswordType::class,
-                'options' => [
+                'constraints' => [
+                    $this->getLengthConstraint($maxLength, $minLength),
+                ],
+                'required' => true,
+                'first_options' => [
+                    'label' => $this->trans('New password', [], 'Admin.Advparameters.Feature'),
+                    'help' => $this->trans(
+                        'Password should be at least %num% characters long.',
+                        [
+                            '%num%' => 8,
+                        ],
+                        'Admin.Advparameters.Help'
+                    ),
                     'attr' => [
                         'data-minscore' => $minScore,
                         'data-minlength' => $minLength,
                         'data-maxlength' => $maxLength,
                     ],
                 ],
-                'constraints' => [
-                    new Length(
-                        [
-                            'max' => $maxLength,
-                            'maxMessage' => $this->getMaxLengthValidationMessage($maxLength),
-                            'min' => $minLength,
-                            'minMessage' => $this->getMinLengthValidationMessage($minLength),
-                        ]
-                    ),
+                'second_options' => [
+                    'label' => $this->trans('Confirm password', [], 'Admin.Advparameters.Feature'),
+                    'help' => '',
+                    'attr' => [
+                        'data-invalid-password' => $this->trans(
+                            'The confirm password doesn\'t match.',
```

Comment:
```suggestion
                            'The confirmation password doesn\'t match.',
```
In the label it's a verb but here it has to be "confirmation". Thank you :)

Reply:
@Julievrz It's fixed.